charmcraft.yaml

```diff
@@ -153,7 +153,6 @@ config:
         will be used if this config option is set to an empty value.
       type: string
     rewrite-enabled:
-      default: false
```

Comment:
Is removing the default intended?
Since we generally want to have sensible defaults.